docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs

```diff
@@ -88,5 +92,92 @@ public static void IidChangePointDetectorTransform()
             // 7       0       7.00    0.50    0.00
             // 7       0       7.00    0.50    0.00
         }
+
+        // This example creates a time series (list of Data with the i-th element corresponding to the i-th time slot). 
```

Comment:
Sorry, I was just mentioning that everything we add to this file will show up verbatim in the docs site on the page for IidChangePointDetector.  Since this adds a second sample to the file, we now have two samples on the docs site - just checking if that is what we actually want.

---
In reply to: [237313709](https://github.com/dotnet/machinelearning/pull/1762#discussion_r237313709) [](ancestors = 237313709)

Reply:
I'll sync offline with you on this.

---
In reply to: [237596871](https://github.com/dotnet/machinelearning/pull/1762#discussion_r237596871) [](ancestors = 237596871,237313709)